packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -163,11 +163,19 @@ Before you can delete an attribute, you first need to remove the mapping for tha
 
   1. Go to the **Provisioning** tab. Under the **Settings** section, click **To App**.
 
-      ![Displays the Provisioning > To App tab.](/img/oin/scim_check-attributes-1.png "Provisioning to App tab")
+     <div class="three-quarter">
+
+     ![Displays the Provisioning > To App tab.](/img/oin/scim_check-attributes-1.png)
+
+     </div>
 
   1. Scroll to the **Attribute Mappings** section. Look for the attribute that you want to delete and then click the corresponding **X**.
 
-      ![Displays the Attribute Mappings screen.](/img/oin/scim_check-attributes-2.png "Attribute Mappings")
+     <div class="three-quarter">
+
+     ![Displays the Attribute Mappings screen.](/img/oin/scim_check-attributes-2.png)
```

Comment:
I would suggest this image can be completely removed. No need to highlight a delete button.